benchmarks/dynamo/common.py

```diff
@@ -1470,11 +1469,13 @@ def parse_args():
 
 def main(runner, original_dir=None):
     args = parse_args()
+    return maybe_fresh_cache(run, args.cold_start_latency)(runner, args, original_dir)
 
+
+def run(runner, args, original_dir=None):
```

Comment:
does this mean the same cache will be used for multiple models in case you don't run w/ --only arg?

Reply:
Yeah, good catch. It will share same cache.

Today, we always isolate by default, i.e., `--only` is the default behavior. 

Other way to handle is to wrap `torch._dynamo.optimize`

Reply:
its 2 layers right- you first launch the outer script without --only, but with --cold_start

then we need to intercept and only add the cache-wrapper inside the subprocess.  so maybe a good way to do it is make this change

`return maybe_fresh_cache(run, args.cold_start_latency && args.only)(...`

Reply:
Good idea. Done!